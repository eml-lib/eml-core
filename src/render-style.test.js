import renderStyle from './render-style';

it('Smthng', () => {
	expect(
		renderStyle({
			width: 100,
			height: '100px',
			backgroundColor: 'red'
		})
	).toMatchSnapshot();
});