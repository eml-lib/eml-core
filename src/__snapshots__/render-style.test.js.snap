// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Smthng 1`] = `"width: 100px; height: 100px; background-color: red"`;
